lib/services/flock.rb

```diff
@@ -1,12 +1,14 @@
 class Service::Flock < Service::Base
   title 'Flock'
 
-  string :url, :placeholder => 'Incoming Webhook URL',
-                 :label => 'Flock Incoming Webhook URL. <br />' \
-                   'To create an incoming webhook, go to your Flock admin panel and switch to "Webhooks" tab'
+  string :url, :placeholder => 'Flock Webhook URL',
+                 :label => 'Go to the <a href="https://apps.flock.co/crashlytics">Flock App Store</a> and install the Crashlytics app. <br />' \
```

Comment:
Hey @ayjindal I started looking at pulling this in, but I can't get to this URL -- can you confirm if it has changed and what the new URL should be?